### PR TITLE
Up to Production 20 Maret 2023 - Office Service Helper

### PR DESCRIPTION
**Pull request by fajarhikmal214**
## Overview
- Up to Production 20 Maret 2023 - Office Service Helper

## Evidence:
project: SIDEBAR
title: Up to Production 20 Maret 2023 - Office Service Helper
participants: @fajarhikmal214 @azophy @sandisunandar99 @muhamadabduh @samudra-ajri @ftmfauzi 
screenshot: https://user-images.githubusercontent.com/79292118/226291289-f2405e98-9a9c-4a75-95ce-417e62403065.png

### CODE REVIEW

**Review by samudra-ajri — APPROVED** [review]

**Review by ftmfauzi — APPROVED** [review]
LGTM 🚀

File: src/pkg/lang/id.json [hunk 1]
```diff
@@ -3,5 +3,28 @@
 
     "common.file.not.found": "File {{ attribute }} tidak ditemukan.",
 
-    "external.pdf.service.error": "Layanan Generate Footer PDF sedang down. Silakan coba lagi dalam beberapa saat."
+    "external.pdf.service.error": "Layanan Generate Footer PDF sedang down. Silakan coba lagi dalam beberapa saat.",
```
Comment (sandisunandar99) [inline src/pkg/lang/id.json:6]: #tipis2 
mungkin redaksinya bisa di rubah jangan "sedang down" ya. misalkan jadi "sedang ada gangguan" atau "sedang dalam gangguan" biar lebih manusiawi aja,, 😄

Comment (sandisunandar99) [reply src/pkg/lang/id.json:6]: eh btw yang bawah nya juga mungkin ya.. yg ada kata2 "down"

Comment (fajarhikmal214) [reply src/pkg/lang/id.json:6]: https://github.com/jabardigitalservice/office-service-helper/pull/23 disini kang

**Review by muhamadabduh — APPROVED** [review]

File: src/pkg/lang/id.json (hunk 1 above)
Comment (sandisunandar99) [reply src/pkg/lang/id.json:6]: okay ,,,@fajarhikmal214

**Review by sandisunandar99 — APPROVED** [review]
Gasss

**Pull request merged**